app/views/prompts/edit.html.erb

```diff
@@ -9,6 +9,15 @@
 <!--/subnav-->
 
 <!--main content-->
+
+<% unless @prompt.errors.empty? %>
```

Comment:
You can leave out this block, since `error_messages_for` already says "We couldn't save this X because".

Reply:
I removed the heading. I also added the "Required information" element to the prompts form to let users know which fields are required, and moved the "Required information" element and the error messages into the actual form tag to get the correct styling.